Update falsepositive.list

erfanosky opened the pull request:
## Domain/URL/IP(s) where you have found the Phishing:
<!-- Required. Use Back ticks. -->
`osky.com.au`

## Describe the issue
<!-- Be as clear as possible: nobody can read your mind, and nobody is looking at your issue over your shoulder. -->
Our site was hacked through WordPress plugin vulnerability and the attacker host phishing form.

We've fixed the issue :
1. Server has been rebuilt from scratch
2. Site has been refresh with clean data
3. All component of the server has been updated and automated ugprade has been implemented.

spirillen commented:
Hey @erfanosky I did notice https://github.com/mitchellkrogza/Phishing.Database/issues/804 and I most admit, my batteries is a bit low to dig into this, so can you provide me with some details about how you spotted/confirmed the hack on your site and which add-on was used to penetrate the WP security?

erfanosky commented:
Hi @spirillen, thanks for looking into this.

We discovered a malicious page redirecting users to a fake Chase bank login. Although the specific compromised plugin is unclear, as a precautionary measure, we rebuilt the website and reinstalled all plugins sourced from the WordPress repository, ensuring each is updated to the latest version.

Additionally, we moved to a new server on AWS, having previously been hosted on shared hosting. To mitigate potential future issues, we implemented Cloudflare to restrict public access to the /wp-admin/ area, allowing only whitelisted IPs to access the WordPress admin dashboard.

Hope this helps but let me know if you require further information.

Regards,
Erfan

spirillen commented:
Hi @erfanosky 

A few friendly meant notes, in accordance to my own privacy aware project.

> new server on AWS

Not privacy friendly as they do log PII data

> we implemented Cloudflare

No, you removed al security regarding encryption and right to privacy. Did you read the part of the license where CF are telling you they are DECRYPTING the connection and using a faked ssl between your server and your visitors, all to be able to cache (the PII data) and then maybe re-encrypt the connection between CF and Amazon?

This removes any trust to your domain, as the connection is insecure. also known as [MITM](#mitm) attack.

> to restrict public access to the /wp-admin/ area

That one is a very simple rule to setup on your server, and the only place to do it, and there are plenty of guides out there for what web server you might use

```nginx

/wp-admin/ {
    allow IP1 ip2 etc
    deny
}
```

No need to give the trust in your domain with crimeflare


## MITM

- [Man-in-the-middle attack](https://en.wikipedia.org/wiki/Man-in-the-middle_attack), a computer networking attack
- [Meet-in-the-middle attack](https://en.wikipedia.org/wiki/Meet-in-the-middle_attack), a cryptographic attack
